packages/oats-runtime/src/make.ts

```diff
@@ -198,10 +211,31 @@ export function makeString(
   };
 }
 
-export function makeNumber(min?: number, max?: number): Maker<number, number> {
-  return (value: any) => {
-    if (typeof value !== 'number') {
-      return getErrorWithValueMsg('expected a number', value);
+export function makeNumber(min?: number, max?: number) {
+  return makeFloatOrInteger(false, min, max);
+}
+export function makeInteger(min?: number, max?: number) {
```

Comment:
separating integers from numbers seems to be the only change now that might break things. Although it is more of a bug fix.

Reply:
> and breaking places where accidentally integer is used for type where number should be used (feels like a bug fix).

yes, good point - it could've been used accidentally (and worked well until this release), so it'll be a breaking change after all